smoketest/configs/bgp-azure-ipsec-gateway

```diff
@@ -267,21 +267,20 @@ system {
     }
     domain-name vyos.net
     flow-accounting {
-        interface eth1
-        interface vti31
-        interface vti32
-        interface vti41
-        interface vti42
-        interface vti51
-        interface vti52
         netflow {
+            interface eth1
+            interface vti31
+            interface vti32
+            interface vti41
+            interface vti42
+            interface vti51
+            interface vti52
```

Comment:
You should not change this file.
The original config from `/* Release version: 1.2.5 */` and all migration scripts will migrate this to the new CLI.
The original config file should be the same.

Reply:
I'll revert these files, thank you.